pandas/tests/frame/test_api.py

```diff
@@ -288,6 +288,22 @@ def test_sequence_like_with_categorical(self):
         for c, col in df.items():
             str(s)
 
+    def test_itertuples_fallback_to_regular_tuples(self):
```

Comment:
can you move this next to the other itertuples tests

Reply:
Done